src/index.css

```diff
@@ -0,0 +1,33 @@
+body {
+  background-color: aquamarine;
+  box-sizing: border-box;
+}
+
+.calc-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+}
```

Comment:
[OPTIONAL] Great job :clap: your calculator width is too much please add `width: 400px` in this `calc-container` then its look better :+1: